aws/scripts/sh/sandbox_installation.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo #### Install Software
+n install ${NODE_VERSION:?Node.js version is not set}
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)